Cert pinning is failing on 'L' preview

```
3109-3164/com.automattic.simplenote.debug E/Simperium.WebSocket﹕ Error: com.koushikdutta.async.http.ConnectionClosedException: connection closed before response completed.
    com.koushikdutta.async.http.ConnectionClosedException: connection closed before response completed.
            at com.koushikdutta.async.http.AsyncHttpResponseImpl$3.onCompleted(AsyncHttpResponseImpl.java:95)
            at com.koushikdutta.async.AsyncSSLSocketWrapper.report(AsyncSSLSocketWrapper.java:394)
            at com.koushikdutta.async.AsyncSSLSocketWrapper.handleResult(AsyncSSLSocketWrapper.java:253)
            at com.koushikdutta.async.AsyncSSLSocketWrapper.write(AsyncSSLSocketWrapper.java:368)
            at com.koushikdutta.async.Util$7.onWriteable(Util.java:192)
            at com.koushikdutta.async.Util.writeAll(Util.java:199)
            at com.koushikdutta.async.Util.writeAll(Util.java:207)
            at com.koushikdutta.async.http.AsyncHttpResponseImpl.setSocket(AsyncHttpResponseImpl.java:66)
            at com.koushikdutta.async.http.AsyncHttpClient$3.onConnectCompleted(AsyncHttpClient.java:370)
            at com.koushikdutta.async.http.AsyncSSLSocketMiddleware$1.onConnectCompleted(AsyncSSLSocketMiddleware.java:52)
            at com.koushikdutta.async.AsyncServer.runLoop(AsyncServer.java:783)
            at com.koushikdutta.async.AsyncServer.run(AsyncServer.java:600)
            at com.koushikdutta.async.AsyncServer.access$700(AsyncServer.java:37)
            at com.koushikdutta.async.AsyncServer$13.run(AsyncServer.java:549)
     Caused by: com.koushikdutta.async.AsyncSSLException: Peer not trusted by any of the system trust managers.
            at com.koushikdutta.async.AsyncSSLSocketWrapper.handleResult(AsyncSSLSocketWrapper.java:252)
            at com.koushikdutta.async.AsyncSSLSocketWrapper.write(AsyncSSLSocketWrapper.java:368)
            at com.koushikdutta.async.Util$7.onWriteable(Util.java:192)
            at com.koushikdutta.async.Util.writeAll(Util.java:199)
            at com.koushikdutta.async.Util.writeAll(Util.java:207)
            at com.koushikdutta.async.http.AsyncHttpResponseImpl.setSocket(AsyncHttpResponseImpl.java:66)
            at com.koushikdutta.async.http.AsyncHttpClient$3.onConnectCompleted(AsyncHttpClient.java:370)
            at com.koushikdutta.async.http.AsyncSSLSocketMiddleware$1.onConnectCompleted(AsyncSSLSocketMiddleware.java:52)
            at com.koushikdutta.async.AsyncServer.runLoop(AsyncServer.java:783)
            at com.koushikdutta.async.AsyncServer.run(AsyncServer.java:600)
            at com.koushikdutta.async.AsyncServer.access$700(AsyncServer.java:37)
            at com.koushikdutta.async.AsyncServer$13.run(AsyncServer.java:549)
     Caused by: javax.net.ssl.SSLPeerUnverifiedException: No peer certificate
            at com.android.org.conscrypt.SSLNullSession.getPeerCertificates(SSLNullSession.java:104)
            at com.koushikdutta.async.AsyncSSLSocketWrapper.handleResult(AsyncSSLSocketWrapper.java:229)
            at com.koushikdutta.async.AsyncSSLSocketWrapper.write(AsyncSSLSocketWrapper.java:368)
            at com.koushikdutta.async.Util$7.onWriteable(Util.java:192)
            at com.koushikdutta.async.Util.writeAll(Util.java:199)
            at com.koushikdutta.async.Util.writeAll(Util.java:207)
            at com.koushikdutta.async.http.AsyncHttpResponseImpl.setSocket(AsyncHttpResponseImpl.java:66)
            at com.koushikdutta.async.http.AsyncHttpClient$3.onConnectCompleted(AsyncHttpClient.java:370)
            at com.koushikdutta.async.http.AsyncSSLSocketMiddleware$1.onConnectCompleted(AsyncSSLSocketMiddleware.java:52)
            at com.koushikdutta.async.AsyncServer.runLoop(AsyncServer.java:783)
            at com.koushikdutta.async.AsyncServer.run(AsyncServer.java:600)
            at com.koushikdutta.async.AsyncServer.access$700(AsyncServer.java:37)
            at com.koushikdutta.async.AsyncServer$13.run(AsyncServer.java:549)
```

--- a/Simperium/build.gradle
+++ b/Simperium/build.gradle
@@ -24,7 +24,7 @@ android {
     defaultPublishConfig 'clientRelease'
 
     dependencies {
-        compile 'com.koushikdutta.async:androidasync:1.2.6@jar'
+        compile 'com.koushikdutta.async:androidasync:1.+@jar'
         compile 'org.thoughtcrime.ssl.pinning:AndroidPinning:1.0.0'
     }
 

